src/Shouldly/ChuckedAWobbly.cs

```diff
@@ -2,6 +2,7 @@
 
 namespace Shouldly
 {
+    [Obsolete("This class is only kept here for backwards compatibility. Please use ShouldAssertException instead.")]
```

Comment:
I think we can drop this, the class was internal so the only use would have been via reflection meaning no one but us will get build warnings